src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -59,6 +60,8 @@ namespace OrchardCore.Commerce;
 
 public class Startup : StartupBase
 {
+    public override int Order => OrchardCoreCommerceConfigureOrder.Default;
```

Comment:
```suggestion
    public override int Order => OrchardCore.OrchardCoreConstantsConfigureOrder.Default;
```

Reply:
I see this is not necessary because that's the default value of [`StartupBase.Order`](https://github.com/OrchardCMS/OrchardCore/blob/84ae5cb0c8e0ec297a645f32b39695c3ea7003a3/src/OrchardCore/OrchardCore.Abstractions/Modules/StartupBase.cs#L12) anyway. I've deleted this line instead.